fec.config.js

```diff
@@ -1,8 +1,9 @@
 const path = require('path');
+const dependencies = require('./package.json').dependencies;
 
 module.exports = {
   appUrl: ['/staging/starter'],
-  sassPrefix: '.widgetLayout',
+  sassPrefix: '.widgetLayout, .landing',
```

Comment:
@Hyperkid123 we should think of nicer sollution for the style sharing between other apps. Perhaps we could try using JSS in this project :thinking:

Reply:
Sass prefix can be disabled. You can use for example `body` as a sass prefix. But that can open a can of worms. JSS would ensure the styling is scoped regardless of where widgets are used. Alternatively, we can always add the `widgetLayout` class to the dashboard wrappers.